pipeline/run_eval.py

```diff
@@ -164,7 +167,11 @@ def get_sampling_cmd(benchmark, random_seed, extra_eval_args="", extra_arguments
     eval_cmds = [" ".join(eval_cmds[i :: args.num_nodes]) for i in range(args.num_nodes)]
 
     for idx, eval_cmd in enumerate(eval_cmds):
-        extra_sbatch_args = ["--parsable", f"--output={args.output_dir}/slurm_logs_eval{idx}.log"]
+        extra_sbatch_args = [
+            "--parsable",
+            f"--output={args.output_dir}/slurm_logs_eval{idx}.log",
+            f"--nv-meta ml-model.llm-reasoning ",
```

Comment:
This should go as part of the extra_sbatch_args to cluster config and not in the code

Reply:
Done